src/stan/lang/grammars/semantic_actions_def.cpp

```diff
@@ -272,14 +293,16 @@ namespace stan {
                          const std::vector<std::vector<expression> >&) const;
     template void assign_lhs::operator()(fun&, const fun&) const;
     template void assign_lhs::operator()(variable&, const variable&) const;
+    template void assign_lhs::operator()(variable_dims&,
```

Comment:
Do we need this?

Reply:
yes, currently needed for `compount_assigment_r` (operator/=)

Reply:
OK!